core/templates/dev/head/pages/collection_editor/editor_tab/CollectionNodeCreatorDirective.js

```diff
@@ -96,6 +123,7 @@ oppia.directive('collectionNodeCreator', [function() {
         };
 
         $scope.addExploration = function() {
+          cleanExplorationId($scope.newExplorationId);
```

Comment:
Change this to not assign explorationId to scope  and just have cleanExplorationId (or the new convert Name) return a value. Then my suggestion would be to return that to addExplorationToCollection. (I.e.
`addExplorationToCollection(convertTypeAhead($scope.newExplorationId))`